tests/adapter/dbt/tests/adapter/basic/test_incremental.py

```diff
@@ -58,5 +60,30 @@ def test_incremental(self, project):
         assert len(catalog.sources) == 1
 
 
+class BaseIncrementalNotSchemaChange:
+    @pytest.fixture(scope="class")
+    def project_config_update(self):
+        return {"name": "incremental"}
+
+    @pytest.fixture(scope="class")
+    def models(self):
+        return {"incremental_not_schema_change.sql": incremental_not_schema_change_sql}
+
+    def test_incremental_not_schema_change(self, project):
+        # Schema change is not evaluated on first run, so two are needed
+        run_dbt(["run", "--select", "incremental_not_schema_change"])
+        run_result = (
+            run_dbt(["run", "--select", "incremental_not_schema_change"]).results[0].status
+        )
+
+        print(run_result)
+
+        assert run_result == RunStatus.Success
+
+
 class Testincremental(BaseIncremental):
     pass
+
+
+class TestBaseIncrementalNotSchemaChange(BaseIncrementalNotSchemaChange):
```

Comment:
Great!! Can we also add this test in [dbt-snowflake](https://github.com/dbt-labs/dbt-snowflake/blob/main/tests/functional/adapter/test_incremental_unique_id.py) and [dbt-redshift](https://github.com/dbt-labs/dbt-redshift/blob/main/tests/functional/adapter/test_incremental_unique_id.py)?